test/root.status

```diff
@@ -0,0 +1,190 @@
+[$mode==debug]
```

Comment:
I'd rather keep the test status in the respective `.status` files in the test directories (e.g. `parallel.status`).

Reply:
How strongly do you feel about that? IMHO having it in one place (1) will enabled easier auditing of this list (2) expresses that it's a cross-cutting concern.

I could instead move all the other rules to this single place, that also has benefits.

Reply:
Not _that_ strongly 😆. 

> I could instead move all the other rules to this single place, that also has benefits.

I could get behind this. My worry is that at some point other annotations (e.g. `flaky` ) start appearing in `root.status` in addition to e.g. `parallel.status` (where they currently are set) and I'd rather avoid any potential confusion. 

What do @nodejs/testing think?